activerecord/test/cases/adapters/sqlite3/sqlite3_adapter_test.rb

```diff
@@ -689,6 +689,61 @@ def test_strict_strings_by_default_and_false_in_database_yml
         end
       end
 
+      def test_strict_tables_is_false
+        conn = Base.sqlite3_connection(database: ":memory:", adapter: "sqlite3")
+        conn.create_table :testings do |t|
+          t.integer :dummy
+        end
+
+        assert_nothing_raised do
+          conn.exec_query("INSERT INTO testings (dummy) VALUES ('foo')")
+        end
+      end
+
+      def test_strict_tables_is_true_and_strict_table_unspecified
+        SQLite3Adapter.strict_tables = true
+        conn = Base.sqlite3_connection(database: ":memory:", adapter: "sqlite3")
+        conn.create_table :testings do |t|
+          t.integer :dummy
+        end
+
+        assert_nothing_raised do
+          conn.exec_query("INSERT INTO testings (dummy) VALUES ('foo')")
+        end
+
+        SQLite3Adapter.strict_tables = false
```

Comment:
We usually do this style differently to avoid having leaky tests. We should take the current value and set it back at the end like this:

```ruby
def test_strict_tables_is_true_and_strict_table_unspecified
  original_strict_tables = SQLite3Adapter.strict_tables
  SQLite3Adapter.strict_tables = true
  conn = Base.sqlite3_connection(database: ":memory:", adapter: "sqlite3")
  conn.create_table :testings do |t|
     t.integer :dummy
  end

  assert_nothing_raised do
     conn.exec_query("INSERT INTO testings (dummy) VALUES ('foo')")
  end

  ensure
    SQLite3Adapter.strict_tables = original_strict_tables
end
```

All the other tests changing strict tables mode should follow the same pattern.